### DISCUSSION TITLE
Effect.Subscriber protection level in release/1.0

### DISCUSSION THREAD

**Discussion by phoenisis** [pointfreeco/swift-composable-architecture#2325] [Beta]
I just tried to migrate from **prerelease/1.0** to **release/1.0** and I’m encountering an issue with the `Effect.Subscriber` protection level. In **prerelease/1.0**, it had public access.
Is it intended to be internal for the future of the composable architecture, or is it a potentially overlooked declaration?

**Comment by stephencelis** [comment]
@phoenisis The Combine version of `Effect.run` (which was the main public use of `Effect.Subscriber`) was deprecated awhile back and is removed in `release/1.0`. It is recommended to use `Effect`'s Swift concurrency APIs, or `Effect.publisher` if you wish to continue to use Combine.
